zanata-model/src/main/java/org/zanata/model/HLocaleMember.java

```diff
@@ -63,6 +65,11 @@ public HLocaleMember(HPerson person, HLocale supportedLanguage,
         setTranslator(isTranslator);
         setReviewer(isReviewer);
         setCoordinator(isCoordinator);
+        userFriendlyToString =
+                String.format(
+                        "Locale membership(person=%s, isTranslator=%s, isReviewer=%s, isCoordinator=%s)",
+                        person.getName(), isTranslator, isReviewer,
```

Comment:
username is in HAccount. Although HPerson has fetch=eager annotation on account field, you can always override this. I don't want to cause an extra database call if it's not there.